### ISSUE TITLE
No warning for <input type="radio"> descendant of role="menuitemradio"

### ISSUE THREAD

**Issue by HadleyLuker** [validator/validator#2107]
### Spec URL

https://www.w3.org/TR/html-aria/#index-aria-menuitemradio

### Test case

```html
<div role="menu">
  <div role="menuitemradio" aria-checked="false">
    <input type="radio" id="example-radio-id">
    <label for="example-radio-id">Lorem</label>
  </div>
</div>
```

**Comment by sideshowbarker** [comment]
I think this was just a deployment problem: One of the multiple server backends for https://validator.w3.org/nu/ had a stale version of the checker deployed, which didn’t include fixes I committed a couple weeks ago. But I’ve updated it now — and so when you check https://validator.w3.org/nu/?doc=data%3Atext%2Fhtml%3Bcharset%3Dutf-8%3Bbase64%2CPCFET0NUWVBFIGh0bWw%2BPGh0bWwgbGFuZz0iIj48dGl0bGU%2BVGVzdDwvdGl0bGU%2BDQo8ZGl2IHJvbGU9Im1lbnUiPg0KICA8ZGl2IHJvbGU9Im1lbnVpdGVtcmFkaW8iIGFyaWEtY2hlY2tlZD0iZmFsc2UiPg0KICAgIDxpbnB1dCB0eXBlPSJyYWRpbyIgaWQ9ImV4YW1wbGUtcmFkaW8taWQiPg0KICAgIDxsYWJlbCBmb3I9ImV4YW1wbGUtcmFkaW8taWQiPkxvcmVtPC9sYWJlbD4NCiAgPC9kaXY%2BDQo8L2Rpdj4%3D now, what you should consistently see is these two errors:

> Error: The element input must not appear as a descendant of an element with the attribute role=menuitemradio.
> Error: The element label must not appear as a descendant of an element with the attribute role=menuitemradio.

So I’m going ahead and closing this. But if the issue is actually that we should be reporting a warning in addition to or instead of either or both of those, please post a comment stating exactly what’s expected, and we could re-open this.
